client/src/components/BizCard_Map/BizCard_Map.jsx

```diff
@@ -0,0 +1,57 @@
+import React, {useState } from 'react';
+import { GoogleMap, Marker, InfoWindow } from '@react-google-maps/api';
+import flag from '../../images/hopper_flag.png'; 
+
+
+function BizCard_Map({props}){
+    //[we need update here]
+    //shop data is in the props. We can plot the spot
+    //by accessing props.lat and props.lng when  backend modify data. 
+    //for now, only Ebi-chan has location property
+    console.log("props",props)
```

Comment:
please remove all your console.logs before pushing, unless you think it's important for the other teammates to have it too!

Reply:
Thx! I removed comment!